### DISCUSSION TITLE
帖子保住了（？）

### DISCUSSION THREAD

**Discussion by Yizakl** [share121/inter-knot#5510] [公告]
昨天晚上
@myitian 大佬已把所有讨论贴保存到互联网档案馆
https://web.archive.org/web/20241001085454/https://github.com/share121/inter-knot/discussions/

**Comment by Faethwith** [comment]
嗯呢呢？（啊？）

**Comment by cyq20091029** [comment]
求科普，互联网博物馆是什么？
能不能介绍一下？

**Reply by Yizakl** [reply]
抱歉
应为**互联网档案馆**
已更正

**Comment by cyq20091029** [comment]
为啥链接打不开啊<img src="https://upload-bbs.mihoyo.com/upload/2022/11/14/d3f9322fcc499df3fca778e8c366f34d_4195564687951765835.png" alt="阿姬疑问" width="75" >

**Reply by Yizakl** [reply]
打不开可能需要七根棍子

**Comment by KELERSAMA** [comment]
ewq 佬
